src/webui/www/private/scripts/client.js

```diff
@@ -52,10 +52,7 @@ window.qBittorrent.Client ??= (() => {
         window.qBittorrent.Cache.qbtVersion.init();
     };
 
-    const closeWindow = (windowID) => {
-        const window = document.getElementById(windowID);
-        if (!window)
-            return;
+    const closeWindow = (window) => {
```

Comment:
"Esc" hotkey no longer works in new feed and edit feed windows (all that i tested).

Reply:
Thank you! It is fixed now.